docs/content/preview/quick-start/build-apps/java/ysql-sdyb.md

```diff
@@ -0,0 +1,318 @@
+        return new NamedParameterJdbcTemplate(dataSource);
+    }
+    
+    @Bean
+    TransactionManager transactionManager(DataSource dataSource) {                     
+        return new YugabyteTransactionManager(dataSource);
+    }
+
+}
+```
+
+2. There are a number of options that can be customized in the properties file located at src/main/resources/application.properties. Given YSQL's compatibility with the PostgreSQL language, the spring.jpa.database property is set to POSTGRESQL and the spring.datasource.url is set to the YSQL JDBC URL: jdbc:postgresql://localhost:5433/yugabyte.
+
+```java
+
+# ---------------------
+# JPA/Hibernate config.
+spring.jpa.database=POSTGRESQL
+
+# The SQL dialect makes Hibernate generate better SQL for the chosen database.
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+
+# Hibernate ddl auto (create, create-drop, validate, update).
+spring.jpa.show-sql=true
+spring.jpa.generate-ddl=true
+spring.jpa.hibernate.ddl-auto=create
+
+# -------------------
+# Data-source config.
+spring.sql.init.platform=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5433/yugabyte
+spring.datasource.username=yugabyte
+spring.datasource.password=
+
+# HikariCP config (pool size, default isolation level).
+spring.datasource.type=com.zaxxer.hikari.HikariDataSource
+spring.datasource.hikari.transactionIsolation=TRANSACTION_SERIALIZABLE
+
+```
+
+3. Create Spring Boot Application runner for performing read and write against YugabyteDB Cluster.
+
+```java
+package com.yugabyte.sdyb.sample;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+@SpringBootApplication
+public class DemoApplication implements CommandLineRunner {
+ 
+ @Autowired
+ JdbcTemplate jdbcTemplate;
+ 
+ @Autowired
+ EmployeeRepository customerRepository;
+
+ public static void main(String[] args) {
+  SpringApplication.run(DemoApplication.class, args);
+ }
+
+ @Override
+ public void run(String... args) throws Exception {
+  
+  System.out.println("Connected to the YugabyteDB server successfully.");
+  jdbcTemplate.execute("DROP TABLE IF EXISTS employee");
+  jdbcTemplate.execute("CREATE TABLE IF NOT EXISTS employee" +
+              "  (id text primary key, name varchar, email varchar)");
+  System.out.println("Created table employee");
+  
+  
+  Employee customer = new Employee("sl1",
+                "User One",
+                "user@one.com");
+  
+  customerRepository.save(customer);
+  
+  Employee customerFromDB = null;
+  customerFromDB = customerRepository.findByEmail("user@one.com");
+  
+  System.out.println(String.format("Query returned: name = %s, email = %s",
+    customerFromDB.getName(), customerFromDB.getEmail()));
+  
+
```

Comment:
indenting and spacing of these last 2 brackets is wonky

Reply:
fixed.